scalableminds/webknossos
# Allow to import NMLs which were created for different datasets

### URL of deployed dev instance (used for testing):
- https://allowdifferentdatasetnml.webknossos.xyz

### Steps to test:
- create a tracing with some trees and download these as NML
- change the "experiment.name" parameter within the NML
- import the NML via dnd
- a modal should ask whether you are sure to import the nml

### Issues:
- fixes #3690
- fixes #3692

------
